### ISSUE TITLE
Play button configuration of Media Config in adaptive host config is not getting used

### ISSUE THREAD

**Issue by mathurvarun84** [microsoft/AdaptiveCards#3099]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* Android
* iOS
* JavaScript
* 

# Author or host

Are you an author (like sending something to Outlook), or a host that is rendering your own cards?
The host is rendering own cards

If you're an author, who are you sending cards to?

# Version of SDK
We are using 1.1.3 of adaptive cards

What version are you using? Ex: NuGet 1.0.2, or latest master, etc...

npm latest

# Issue

We are trying to use adaptive card host config to put the playbutton image for our video cards.
We had given the configuration as following

media:{
playbutton: `<some external url>`
}

However, even after giving this the card doesn't display the playbutton as we specified.

When I looked at following code
https://github.com/microsoft/AdaptiveCards/blob/master/source/nodejs/adaptivecards/src/host-config.ts

I found that the constructor is not even taking playbutton as input.

Is it a bug? Can you please let us know what needs to be done here.

Explain your issue. If you just have a question, please post [on StackOverflow instead](https://stackoverflow.com/questions/tagged/adaptive-cards).

**Comment by dclaux** [comment]
The JS renderer doesn't use HostConfig to customize the play button. There are too many ways an application might want to style the button for a simple image to be enough.

To customiize the play button, modify the Adaptive Card CSS, namely the `ac-media-playButton` and `ac-media-playButton-arrow` classes.

**Comment by andrewleader** [comment]
Triage notes: We discussed in triage, the play icon is an [official host config feature](https://docs.microsoft.com/en-us/adaptive-cards/rendering-cards/host-config#schema-mediaconfig) and should be supported on all platforms. Additional extensibility on top of that (like CSS extensibility) is very valuable, but official host config features should be supported as a bare minimum.

@almedina-ms , @nesalang , can you investigate whether iOS/Android correctly uses the play image icon from host config?

**Comment by nesalang** [comment]
on iOS, I found a regression. iOS renderer do use play icon from host config, but the regression caused not to remove default play icon. So two icon will get overlapped, and it appears the icon from host config not being used.

**Comment by cmevawala** [comment]
@andrewleader : Any updates on this ?

**Comment by andrewleader** [comment]
@cmevawala no, no updates to share yet. Are you blocked on this issue? If this is a blocking important issue, please let us know!

**Comment by shalinijoshi19** [comment]
@almedina-ms can you double check the story on Android as well? Thanks

**Comment by almedina-ms** [comment]
@shalinijoshi19 this works in android

![Screenshot_2019-07-11-12-47-07](https://user-images.githubusercontent.com/35784165/61076677-a5ffae00-a3d1-11e9-8624-86e973d03e79.png)

**Comment by almedina-ms** [comment]
@nesalang and @dclaux , is there any update for this?

**Comment by dclaux** [comment]
Not sure what kind of update I could give you on the JS side.

**Comment by nesalang** [comment]
@almedina-ms 
for iOS, this issue has been resolved as stated in the jun 24th comment.

**Comment by almedina-ms** [comment]
> 
> 
> Triage notes: We discussed in triage, the play icon is an [official host config feature](https://docs.microsoft.com/en-us/adaptive-cards/rendering-cards/host-config#schema-mediaconfig) and should be supported on all platforms. Additional extensibility on top of that (like CSS extensibility) is very valuable, but official host config features should be supported as a bare minimum.
> 
> @almedina-ms , @nesalang , can you investigate whether iOS/Android correctly uses the play image icon from host config?

@dclaux , regarding @andrewleader  's comment, if the issue is still there then there's no update but as Android and iOS don't have this issue I could assign the issue to you?

**Comment by dclaux** [comment]
You can assign to me, but I'm going to be honest, I will treat it as extremely low priority and might even never get to it.

Just as another reminder: HostConfig was designed to make it possible for one host to display cards like another host. Aside from the Designer (and the Visualizer before it) that capability has never been used. Besides, HostConfig does not support customization of buttons and inputs, which are most likely used way way more than Media elements.

Using an image for the Play button isn't a great solution because:
- If the image used is low(ish) resolution, it will be aliased when rendered on high dpi screens and will therefore look bad
- If the image used is high resolution, then it will consume more bandwidth than necessary on lower dpi machines and will be shrunk on screen yielding to smoothed edges which again won't look great (although not as bad as when the icon is blown up)

The JS implementation is dpi aware and the play button always looks great. It is customizable in CSS like inputs and buttons, and it emulates the look and feel you get by default in Chrome and now Chromium Edge, leading to better consistency.

For all these reasons (which I already gave when implementing support for the Media element, and which were accepted as valid enough) adding support for Play button in HostConfig is very, very low priority and I would go as far as saying unnecessary.

**Comment by shalinijoshi19** [comment]
This should be fixed in what i have gleaned of the issue; Taking note of David's point of view though labeling it lower priority to revisit/loadbalance.

**Comment by paulcam206** [comment]
This is not currently on our roadmap, please use the recommended workaround.
